src/main/java/teammates/common/util/Const.java

```diff
@@ -583,6 +583,8 @@ private Const() {
         public static final String INSTRUCTOR_EMAIL = "instructoremail";
         public static final String INSTRUCTOR_INSTITUTION = "instructorinstitution";
         public static final String INSTRUCTOR_NAME = "instructorname";
+        public static final String NAME = "name";
+        public static final String EMAIL = "email";
```

Comment:
Maybe just me, but I fail to see where these two are used.

Reply:
Nope its not you! Removed